src/Profiles/Fec/DataPages/ANTPLUS_FecTrackResistanceDataPage.cpp

```diff
@@ -0,0 +1,31 @@
+#include <Profiles/Fec/DataPages/ANTPLUS_FecTrackResistanceDataPage.h>
+#include <Profiles/Fec/ANTPLUS_FecPrivateDefines.h>
+#include <ANTPLUS_PrivateDefines.h>
+
+// TODO this is a mixture of datapages

```

Comment:
what do you mean by this comment? This looks like one page, although yes there is a TODO as last field is missing.